api/src/main/resources/applicationContext-service.xml

```diff
@@ -678,4 +678,9 @@
 		<context:exclude-filter type="custom" expression="org.openmrs.annotation.OpenmrsProfileExcludeFilter"/>
 	</context:component-scan>
 
+	<context:property-placeholder
+		location="file:${OPENMRS_APPLICATION_DATA_DIRECTORY}/openmrs-runtime.properties"
+		system-properties-mode="OVERRIDE"
+		ignore-resource-not-found="true" />
```

Comment:
Could we please also include hibernate.default.properties and have a simple BaseContextSensitiveTest that uses e.g. @Value annotation or Environment to test if properties are loaded correctly?

For including extra locations with ordering you would probably need to define bean directly instead of using context:property-placeholder.

```
<bean class="org.springframework.context.support.PropertySourcesPlaceholderConfigurer">
  <property name="location">
  <list>
  <value>classpath:hibernate.default.properties</value>
  <value>file:${OPENMRS_APPLICATION_DATA_DIRECTORY}/openmrs-runtime.properties</value>
  </list>
  </property>
  ...
</bean>
```